doc/source/data/examples/batch_training.ipynb

```diff
@@ -585,9 +588,11 @@
     "    \n",
     "    # Evaluate.\n",
     "    error = sklearn.metrics.mean_absolute_error(test_y, pred_y)\n",
+    "    if error is None:\n",
+    "        error = 10000.0\n",
     "    \n",
     "    # Assemble return as a pandas dataframe.\n",
-    "    return_df = pd.DataFrame({'model': [model], 'error': [error]})\n",
+    "    return_df = pd.DataFrame({'model': [str(model)], 'error': [error]})\n",
```

Comment:
why are we returning a string? the model would be better imo, as it's fitted and ready to be used/inspected

Reply:
I was worried about space.  I guess these sklearn binary models are small enough.  Will make the change, thanks.